modules/dragster/src/main/ts/ephox/dragster/detect/Blocker.ts

```diff
@@ -21,11 +21,12 @@ export const Blocker = (options: Partial<BlockerOptions>): Blocker => {
   const div = SugarElement.fromTag('div');
   Attribute.set(div, 'role', 'presentation');
   Css.setAll(div, {
-    position: 'fixed',
-    left: '0px',
-    top: '0px',
-    width: '100%',
-    height: '100%'
+    'position': 'fixed',
+    'left': '0px',
+    'top': '0px',
+    'width': '100%',
+    'height': '100%',
+    'z-index': '10000'
```

Comment:
Are we confident that this value is high enough. I looks like the floating UI In inline mode is hidden while resizing can we add some comment what it's higher than. I guess at some point we should have these things defined. It looks like the resize corners have indexes around 1000 and that the current version has a bug where the blocker has lower z-index that than so resizing stops if you hit that corner I wonder if there are other things like that.